test/e2e/app-dir/next-after/app/page.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Page() {
+  return <p>hello world</p>
```

Comment:
Here it might be worth adding a `Suspense` with both this `Page` and an async child component calling to `after()`